core/src/memory/resources/terminal/malloc_resource.cpp

```diff
@@ -0,0 +1,31 @@
+#include <memory>
+#include <memory_resource>
+
+#include "vecmem/memory/resources/terminal/malloc_resource.hpp"
+
+namespace vecmem::memory::resources::terminal {
+    void * malloc_resource::do_allocate(
+        std::size_t bytes,
+        std::size_t
+    ) {
+        return malloc(bytes);
+    }
+
+    void malloc_resource::do_deallocate(
+        void * p,
+        std::size_t,
+        std::size_t
+    ) {
+        free(p);
+    }
+
+    bool malloc_resource::do_is_equal(
+        const std::pmr::memory_resource & other
+    ) const noexcept {
+        return false;
```

Comment:
Why would two malloc resources never be equal to each other? I would've thought they would always be equal... :confused: